src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -109,12 +109,21 @@ const TableHeader: React.FC<TableHeaderProps> = ({
           </button>
         </div>
         <div className="table-title-container">
-          <div className="private-table-title" style={{ fontSize: "20px" }}>
+          <div
+            className="private-table-title"
+            style={{ fontSize: "16px", fontWeight: 700 }}
+          >
```

Comment:
[TECH-DEBT] This should be moved to our scss and use classes rather than inline `styles`

Reply:
backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/314